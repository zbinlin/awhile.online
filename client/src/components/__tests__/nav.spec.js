"use strict";

/* eslint-env jest */
import { h } from "preact";
import renderer from "react-test-renderer";
import Nav from "../nav.jsx";

it("Nav renders correctly", () => {
    const tree = renderer.create(
        <Nav value={[{name: "首页", value: "/"}, {name: "test", value: null}]} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
