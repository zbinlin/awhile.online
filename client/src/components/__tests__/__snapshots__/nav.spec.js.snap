exports[`test Nav renders correctly 1`] = `
<div
  className="nav">
  <a
    href="/">
    首页
  </a>
  <span>
    test
  </span>
</div>
`;
